static/app/components/events/interfaces/performance/utils.tsx

```diff
@@ -26,12 +22,7 @@ export function getSpanInfoFromTransactionEvent(
 ) {
   const perfEvidenceData = event.perfProblem ?? event?.occurrence?.evidenceData;
   if (!perfEvidenceData) {
-    if (
-      event.issueCategory === IssueCategory.PERFORMANCE &&
-      event.endTimestamp > 1663560000 //  (Sep 19, 2022 onward), Some events could have been missing evidence before EA
```

Comment:
This function is only called when rendering the span evidence section, so the category check is unnecessary. The time check is also unnecessary since those events no longer exist.